File: Python/백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.py
a=int(input())
check=a
for i in range(a):
    x=input()
    l=len(x)
    for j in range(0,l-1):
        if(x[j]!=x[j+1]):
            if(x[j] in x[j+1:]):
                check=check-1
                break      
print(check)

File: C#/백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int check=a;
            for(int i=0; i<a; i++){
                string x=Console.ReadLine();
                int l=x.Length;
                for(int j=0;j<l-1;j++){
                    if(x[j]!=x[j+1]){
                        if(x.Substring(j+1).Contains(x[j])){
                            check-=1;
                            break;
                        }
                    }
                }
            }
            Console.WriteLine(check);
        }
    }
}